modules/src/address.rs

```diff
@@ -1 +1,23 @@
-pub struct AccAddress(Vec<u8>);
+// TODO - remove when ICS message signer changes from AccountId to String
+
+use std::convert::TryFrom;
+
+use anomaly::{BoxError, Context};
+use bech32::FromBase32;
+use bech32::ToBase32;
+
+use tendermint::account::Id as AccountId;
+
+pub fn account_to_string(addr: AccountId) -> Result<String, BoxError> {
```

Comment:
Let's discuss this there. Not sure about the benefit of having a new type since we completely ignore this field in the IBC handlers (other than converting it back and forth).

Reply:
Yes! Since handlers/modules ignore the signer, it is clearly best to remove this field altogether. We can do that as part of #219.